### ISSUE TITLE
On Linux, a new file identical to the one it replaces counts as a change, but it shouldn't.

### ISSUE THREAD

**Issue by styromaniac** [HelloZeroNet/ZeroNet#2575]
I have cron setup on my laptop to download Ngosang's trackers list. I've had it set to every minute for testing purposes. Even when Ngosang's site says his list updated seven hours ago, every single minute ZeroNet thinks the file is updated again. This is a false positive.

**Comment by HelloZeroNet** [comment]
So the file that contains the trackers got downloaded again even if it's not changed or only the site's changed date (in content.json) got modified?

**Comment by styromaniac** [comment]
> So the file that contains the trackers got downloaded again even if it's not changed or only the site's changed date (in content.json) got modified?

Yes, it's going by timestamp for some reason. It sees a different timestamp as meaning a file change.

**Comment by HelloZeroNet** [comment]
Right now if you execute a sign command, then it will sign and publish the site with all the files, then add the current timestamp as modification time to content.json. (ZeroHello displays this timestamp as modification time) 

Even if there was no change in any file. It could be possible to ignore the site sign command if it does not detect any changes in site's files.

**Comment by styromaniac** [comment]
I don't mean content.json, but I think it's the created/last-modified date/time showing on the computer's file manager/browser that makes it think it's a changed file.

**Comment by HelloZeroNet** [comment]
You mean the notification on the web interface in the bottom-right corner?

**Comment by styromaniac** [comment]
> You mean the notification on the web interface in the bottom-right corner?

Yes, with the sign and publish button. That pops up every time the same exact file is written to storage again.
